src/Content/MediaFactory.php

```diff
@@ -65,6 +67,10 @@ public function createOrUpdateMedia(SplFileInfo $file, string $area): Media
                 ->setArea($area);
         }
 
+        if (! $this->mediatypes->contains($file->getExtension())) {
+            die('Not a valid media type.');
```

Comment:
Because `die()` is bad, and it'll get fixed properly.. If I slap an exception on it now, it might stay that way indefinitely. The "media" part needs work, so it'll be handled better then. 😇

Reply:
can you replace it with an \Exception and add @todo? At least for getting trace where it happened :)